src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -45,12 +44,6 @@ class SideNav extends Component {
 
     // Escape early if the item has children.
     if (hasChildren) {
-      // Track expand/collapse.
-      recordEvent({
```

Comment:
@jonathanjnelson did you want to keep this to track the expand events?

Reply:
By removing this code, we'd be removing event tracking for child nav items:

![image](https://user-images.githubusercontent.com/12773166/81217209-7261c600-8f99-11ea-8ffd-893ba74293b4.png)